centraldogma/dogma.py

```diff
@@ -312,3 +314,25 @@ def listener(revision: Revision, value: MyType) -> None:
         )
         watcher.start()
         return watcher
+
+    def merge_files(
+        self,
+        project_name: str,
+        repo_name: str,
+        merge_sources: List[MergeSource],
+        json_paths: List[str] = None,
+        revision: Optional[int] = None,
+    ) -> MergedEntry[T]:
```

Comment:
`T` is only bound to the return type which seems not to tell any useful type information.
How about replacing `T` with `Any` type?
```suggestion
    ) -> MergedEntry[Any]:
```

Reply:
Just removed the generic argument altogether 😅 
Although the java api is defined with a generic type, I'm not foreseeing many changes to the merge feature (and I guess we can make a breaking change if necessary)